server/handlers/project_command_output_handler.go

```diff
@@ -67,7 +77,7 @@ type ProjectCommandOutputHandler interface {
 //go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_resource_cleaner.go ResourceCleaner
 
 type ResourceCleaner interface {
-	CleanUp(pull string)
+	CleanUp(jobContext models.PullContext)
```

Comment:
Shouldn't this be `models.JobContext` ?

Reply:
JobContext is basically PullContext + commitSha https://github.com/lyft/atlantis/blob/852ac4cea04d10b909c4293e3a62278a695b0e53/server/events/models/models.go#L717-L727 
we pass in JobContext to CleanUp() method because there can be more than one revisions in the PR with corresponding plan and apply jobs. So, having pull context instead of job context allows us to perform cleanup for all the jobs in a particular PR.

Reply:
ok, then let's rename the argument to `pullContext`, it will be confusing to pass in `PullContext` but call it `JobContext` while having separate struct called `JobContext`

Reply:
Didn't realize that I had `jobContext` as the argument name! I'll change it to `pullContext`

Reply:
Done!